drivers/flash/nrf_qspi_nor.c

```diff
@@ -403,16 +403,16 @@ static inline void qspi_fill_init_struct(nrfx_qspi_config_t *initstruct)
 #endif
 
 	/* Configure Protocol interface */
-#if DT_INST_NODE_HAS_PROP(0, readoc_enum)
+#if DT_INST_NODE_HAS_PROP(0, readoc)
```

Comment:
OK, this change is absolutely necessary, and is a bug introduced by #24378 if it wasn't broken already.  The effect is that the devicetree `readoc` and `writeoc` properties are being ignored.

Possibly we should try to get this into 2.4.0.  @ioannisg @carlescufi

Reply:
Will need a review from @nvlsianpu  ,right?

Reply:
Presumably.  Why doesn't github automatically add maintainers?  nvm.